framework/test/unit/modules/interoperability/base_cross_chain_update_command.spec.ts

```diff
@@ -728,84 +763,62 @@ describe('BaseCrossChainUpdateCommand', () => {
 		});
+	});
 
-		it('should update validators when activeValidatorsUpdate is not empty', async () => {
-			executeContext = createTransactionContext({
-				chainID,
-				stateStore,
-				transaction: new Transaction({
-					...defaultTransaction,
-					command: command.name,
-					params: codec.encode(crossChainUpdateTransactionParams, {
-						...params,
-						inboxUpdate: {
-							...params.inboxUpdate,
-							activeValidatorsUpdate: [
-								{ bftWeight: BigInt(1), blsKey: utils.getRandomBytes(BLS_PUBLIC_KEY_LENGTH) },
-							],
-						},
-					}),
-				}),
-			}).createCommandExecuteContext(command.schema);
-
-			await expect(command['executeCommon'](executeContext, true)).resolves.toEqual([
-				expect.toBeArrayOfSize(params.inboxUpdate.crossChainMessages.length),
-				true,
-			]);
-			expect(internalMethod.updateValidators).toHaveBeenCalledTimes(1);
-		});
+	describe('afterExecuteCommon', () => {
+		let executeContext: CommandExecuteContext<CrossChainUpdateTransactionParams>;
 
-		it('should update validators when certificate threshold is different', async () => {
+		beforeEach(() => {
 			executeContext = createTransactionContext({
-				chainID,
-				stateStore,
 				transaction: new Transaction({
 					...defaultTransaction,
 					command: command.name,
 					params: codec.encode(crossChainUpdateTransactionParams, {
 						...params,
 						inboxUpdate: {
 							...params.inboxUpdate,
-							certificateThreshold: BigInt(1),
 						},
+						activeValidatorsUpdate: {
+							blsKeysUpdate: [utils.getRandomBytes(48)],
+							bftWeightsUpdate: [BigInt(1), BigInt(3), BigInt(4), BigInt(3)],
+							bftWeightsUpdateBitmap: Buffer.from([1]),
+						},
+						sendingChainID: defaultSendingChainID,
+						certificateThreshold: BigInt(20),
+						certificate,
 					}),
 				}),
 			}).createCommandExecuteContext(command.schema);
 
-			await expect(command['executeCommon'](executeContext, true)).resolves.toEqual([
-				expect.toBeArrayOfSize(params.inboxUpdate.crossChainMessages.length),
-				true,
-			]);
-			expect(internalMethod.updateValidators).toHaveBeenCalledTimes(1);
+			const chainValidatorsStore = command['stores'].get(ChainValidatorsStore);
+			jest.spyOn(chainValidatorsStore, 'get').mockResolvedValue({
+				certificateThreshold: BigInt(10),
+			} as any);
+
+			const channelDataStore = command['stores'].get(ChannelDataStore);
+			jest.spyOn(channelDataStore, 'get').mockResolvedValue({
+				inbox: {
+					size: 1,
+				} as any,
+			} as any);
 		});
 
-		it('should update certificate when certificate is not empty', async () => {
-			await expect(command['executeCommon'](executeContext, true)).resolves.toEqual([
-				expect.toBeArrayOfSize(params.inboxUpdate.crossChainMessages.length),
-				true,
-			]);
-			expect(internalMethod.updateCertificate).toHaveBeenCalledTimes(1);
-		});
+		it('should update validators if activeValidatorsUpdate is not empty', async () => {
```

Comment:
I think there should be separate test cases to cover, 
`if len(ccu.params.certificate) > 0 updateCertificate(ccu)`
`if ccu.params.inboxUpdate is not empty and len(ccu.params.certificate) > 0: updatePartnerChainOutboxRoot(ccu)`